x-pack/solutions/search/plugins/search_indices/public/components/indices/details_search_example.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { SearchHit } from '@kbn/es-types';
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { docLinks } from '../../../common/doc_links';
+import { Mappings } from '../../types';
+
+import { CodeBox } from '../code_box/code_box';
+import { useSearchCodeExamples } from '../../hooks/use_search_code_examples';
+
+export interface IndexSearchExampleProps {
+  indexName: string;
+  documents: SearchHit[];
+  mappings?: Mappings;
+  navigateToPlayground: () => void;
+}
+
+export const IndexSearchExample = ({
+  indexName,
+  mappings,
+  navigateToPlayground,
+}: IndexSearchExampleProps) => {
+  const codeExamples = useSearchCodeExamples(indexName, mappings);
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem grow={2}>
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.searchIndices.indexDetail.searchExample.title"
+              defaultMessage="Search your data"
+            />
+          </h3>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.searchIndices.indexDetail.searchExample.description"
+              defaultMessage="A search query is a request for information about data in Elasticsearch indices or datastreams."
```

Comment:
```suggestion
              defaultMessage="A search query finds relevant matches in your Elasticsearch data based on your chosen search approach."
```

I don't know if we want a docs link here, but we have a nice [search approaches](https://www.elastic.co/docs/solutions/search/search-approaches) overview page if we wanted to :)

Reply:
I'd go with the second one, if linking to docs is overkill